src/targets/decorate-storybook.js

```diff
@@ -51,8 +51,20 @@ function decorateStorybook(storybook) {
     return stories;
   }
 
-  /* eslint no-param-reassign: ["error", { "props": false }] */
-  storybook.storiesOf = storiesOf;
+  // Monkey patch storiesOf to be able to add async/skip methods
+  const descriptor = Object.getOwnPropertyDescriptor(storybook, 'storiesOf');
+  if (descriptor.writable) {
```

Comment:
When the descriptor is not writable? Should we maybe add a small comment to explain?

Reply:
I'm actually not 100% sure, but assumes it's because of some change to how babel transpiles exports. Want me to add that as a speculation?